a3p-integration/proposals/z:acceptance/test/localchain.test.js

```diff
@@ -1,25 +1,33 @@
 /* global setTimeout */
 
-import { agd, evalBundles } from '@agoric/synthetic-chain';
+import {
+  agd,
+  evalBundles,
+  passCoreEvalProposal,
+  GOV1ADDR,
+} from '@agoric/synthetic-chain';
 import test from 'ava';
+import fs from 'node:fs/promises';
+import path from 'node:path';
 import { retryUntilCondition } from '@agoric/client-utils';
+import { encodeAddressHook } from '@agoric/cosmic-proto/address-hooks.js';
 
-const SUBMISSION_DIR = 'localchaintest-submission';
+const SUBMISSION_DIR = 'test/localchaintest-submission';
 
-/** @param {string} path */
-const readPublished = async path => {
+/** @param {string} nodeName */
+const readPublished = async nodeName => {
   const { value } = await agd.query(
     'vstorage',
     'data',
     '--output',
     'json',
-    `published.${path}`,
+    `published.${nodeName}`,
   );
   if (value === '') {
     return undefined;
   }
   const obj = JSON.parse(value);
-  return obj.values[0];
+  return obj.values[obj.values.length - 1];
```

Comment:
took me a while to figure out that you're switching from the 1st to the last. suggest:

```suggestion
  return obj.values.at(-1);
```

Reply:
Thanks, done.